src/main/java/bio/terra/service/filedata/flight/ingest/IngestBuildAndWriteScratchLoadFileStep.java

```diff
@@ -70,8 +79,7 @@ public StepResult doSkippableStep(FlightContext context) {
                         BulkLoadFileModel fileModel =
                             Objects.requireNonNull(
                                 objectMapper.convertValue(fileRefNode, BulkLoadFileModel.class));
-                        int fileKey =
-                            Objects.hash(fileModel.getSourcePath(), fileModel.getTargetPath());
+                        int fileKey = Objects.hash(fileModel.getTargetPath());
```

Comment:
Don't we also want the source file in here?

Reply:
Unfortunately, the `FileModel` of already-ingested files doesn't include the source path. At this point, the file-loading has already happened, and that means that all of the loaded files have different target paths. So, we can use _just_ the target path as the key here.